clients/python/llmengine/model.py

```diff
@@ -115,33 +116,19 @@ def create(
                 of a single worker
 
             per_worker (`int`):
-                The maximum number of concurrent requests that an individual worker can
-                service. Launch automatically scales the number of workers for the endpoint so that
-                each worker is processing ``per_worker`` requests, subject to the limits defined by
-                ``min_workers`` and ``max_workers``
-
-                - If the average number of concurrent requests per worker is lower than
-                ``per_worker``, then the number of workers will be reduced. - Otherwise,
-                if the average number of concurrent requests per worker is higher than
-                ``per_worker``, then the number of workers will be increased to meet the elevated
-                traffic.
-
-                Here is our recommendation for computing ``per_worker``:
-
-                1. Compute ``min_workers`` and ``max_workers`` per your minimum and maximum
-                throughput requirements. 2. Determine a value for the maximum number of
-                concurrent requests in the workload. Divide this number by ``max_workers``. Doing
-                this ensures that the number of workers will "climb" to ``max_workers``.
+                Number of Uvicorn workers per pod. Recommendation is set to 2.
```

Comment:
Might want to hide details of Uvicorn.

Also I don't think this is correct? IIRC `per_worker` is basically a scaling sensitivity parameter. cc @seanshi-scale

Reply:
hmm, this actually decides both # of workers and HPA target concurrency https://github.com/scaleapi/llm-engine/blob/main/model-engine/model_engine_server/infra/gateways/k8s_resource_parser.py#L65
i'll make some updates